dts/bindings/display/ilitek,ili9341.yaml

```diff
@@ -0,0 +1,146 @@
+# Copyright (c) 2018, Jan Van Winkel <jan.van_winkel@dxplore.eu>
+# Copyright (c) 2020, Teslabs Engineering S.L.
+# Copyright (c) 2021, Krivorot Oleg <krivorot.oleg@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: ILI9341 320x240 display controller
+
+compatible: "ilitek,ili9341"
+
+include: ilitek,ili9xxx-common.yaml
+
+properties:
+    pwctrla:
+      type: uint8-array
+      default: [0x39, 0x2c, 0x00, 0x34, 0x02]
+      description:
+        Power control A (PWCTRLA) register value.
+
+    pwctrlb:
+      type: uint8-array
+      default: [0x00, 0x8b, 0x30]
+      description:
+        Power control B (PWCTRLB) register value.
+
+    pwseqctrl:
+      type: uint8-array
+      default: [0x55, 0x01, 0x23, 0x01]
+      description:
+        Power on sequence control (PWSEQCTRL) register value.
+
+    timctrla:
+      type: uint8-array
+      default: [0x84, 0x11, 0x7a]
+      description:
+        Driver timing control A (TIMCTRLA) register value.
+
+    timctrlb:
+      type: uint8-array
+      default: [0x00, 0x00]
+      description:
+        Driver timing control B (TIMCTRLB) register value.
+
+    pumpratio:
```

Comment:
naming should be consistent with datasheet and description field, `pumpratioctrl`?